lib/volt/volt/users.rb

```diff
@@ -114,6 +114,10 @@ def login(username, password)
     end
 
     def logout
+      # Notify the backend so we can remove the user_id from the user's channel
+      UserTasks.logout(user_id: Volt.current_app.cookies._user_id)
```

Comment:
the user_id cookie is encrypted and in this case will get passed as metadata since your calling UserTasks.logout before the actual logout.  So you can probably just remove the arguments.